addons/website/views/snippets/snippets.xml

```diff
@@ -301,11 +301,11 @@
 
     <div data-js="sizing_y"
         data-selector="section, .row > div, .parallax, .s_hr, .s_btn, .carousel-item"
-        data-exclude="section:has(> .carousel), .o_gallery .carousel-item"/>
+        data-exclude="section:has(> .carousel), .o_gallery .carousel-item, .s_company_team_member > div"/>
 
     <div data-js="sizing_x"
         data-selector=".row > div"
-        data-drop-near=".row > div"
+        data-drop-near=".row:not(.s_company_team_member) > div"
```

Comment:
I think felix fixed that in its branch about for, it should simply be `:not(.s_col_no_resize)` here